tests/contrib/scrapy/test_pipelines.py

```diff
@@ -210,6 +212,25 @@ class PipelineJSONSchemaValidator(PipelineTest):
     ]
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 4), reason="mock requires python3.4 or higher"
+)
+def test_validator_from_url():
+    from unittest import mock
+
+    with mock.patch("spidermon.utils.web.get_contents") as mocked_get_contents:
```

Comment:
You have suggested me in other PR, so why you don't use `pytest-mock` and avoid the importing here?